samples/DotNet/Microsoft.Azure.EventHubs/SampleEphReceiver/readme.md

```diff
@@ -1,16 +1,16 @@
 # Receive events with the Event Processor Host in .NET Standard
 
-This sample shows how to write a .NET Core console application that receives a set of events from an event hub by using the **Event Processor Host** library. You can run the solution as-is, replacing the strings with your event hub and storage account values. The sample is also [available as a tutorial](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-dotnet-standard-getstarted-receive-eph).
```

Comment:
This link gets redirected to Track 2 quick start, so re-worded the entire intro to refer to the link as T2 and not T1